### ISSUE TITLE
ERROR: 'charmap' codec can't decode byte 0x81 in position 4429...

### ISSUE THREAD

**Issue by hcadre** [MannLabs/alphabase#181]
**Describe the bug**
Running alphaDIA (one-Click installer for Windows) with the example HeLa test samples and the parameters as described resulted in this error thrown after a few seconds:

![image](https://github.com/MannLabs/alphadia/assets/16349515/f62432cb-20c2-41b7-83d5-30bbb5556613)

Any idea about what went wrong?

**Comment by GeorgWa** [comment]
@hcadre thank you for uploading the fasta. It's not malformed, alphaBase just tries to parse it as utf8 and fails doing so. I have transferred the issue and will close it as soon as we have a fix.

**Comment by LoayJabre** [comment]
Hi - 
I tried re-running everything today, and it worked fine! 

I tried with and without the .fasta and it worked. It must have been something I was doing wrong initially. 

Out of curiosity, is there a column key anywhere for the precursors.tsv file that is outputted after the search? I'm fairly new to this, and it would help a lot if there is a key to help me understand the output a bit better.

Thanks so much for being responsive! 

Loay
